analysis/test/golden_test.sh

```diff
@@ -83,6 +83,40 @@ check_merge() {
   validate "${ACTUAL_MERGE_JSON}"
 }
 
+check_modify() {
+    echo " -- expect StructureModifier gives valid cc.json"
+    ACTUAL_MODIFY_JSON="${INSTALL_DIR}/actual_modify.json"
+    "${CCSH}" modify data/codecharta/tomerge.json --moveFrom=root/src --moveTo=root/bar > "${ACTUAL_MODIFY_JSON}"
+    validate "${ACTUAL_MODIFY_JSON}"
+}
+
+check_sourcecodeparser() {
+    echo " -- expect SourceCodeParser gives valid cc.json"
+    ACTUAL_SCP_JSON="${INSTALL_DIR}/actual_modify.json"
```

Comment:
Should they use the actual_modify.json? Or is this just a name for the file and it doesnt matter?

Reply:
it's just a name, doesn't change anything... but yes it is a strange name for the function (forgot to rename after copy/paste), i will change the names